packages/vtable/src/ts-types/style-define.ts

```diff
@@ -14,7 +14,7 @@ export interface StylePropertyFunctionArg {
   /**有format的话 格式化后或者计算后的值 */
   value: string;
   /**原始值 */
-  dataValue: string;
+  dataValue: string | number;
```

Comment:
![image](https://github.com/VisActor/VTable/assets/3468473/452d626e-361f-475b-a07b-275d4cba0d63)
value and dataValue can define as `FieldData`